DP02_09_Custom_Coadds/DP02_09b_Custom_Coadd_Sources.ipynb

```diff
@@ -345,371 +336,329 @@
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "4c69c185-cea1-4c38-a1ae-41b42719cd7f",
+   "id": "03e56251-fc55-4214-a40d-46d70d0c1f05",
    "metadata": {},
    "outputs": [],
    "source": [
-    "config = CharacterizeImageTask.ConfigClass()\n",
-    "config.psfIterations = 1\n",
-    "charImageTask = CharacterizeImageTask(config=config)"
+    "simpleButler = SimplePipelineExecutor.prep_butler(butler_repo, \n",
+    "                                                  inputs=[inputCollection], \n",
+    "                                                  output=my_outputCollection)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "b6236c37-1cba-4bcf-8a08-7ff6caeb4e59",
+   "id": "0e256101-cfeb-4691-8ce8-ae676c83219d",
    "metadata": {},
    "source": [
-    "Configure the task to detect sources.\n",
-    "Set the threshold that defines what \"detected\" means to be 10 times above the standard deviation (i.e., a signal-to-noise ratio of 10)."
+    "Now check the collection chain associated with your source detection output collection:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "059ea6a4-c0ec-4a96-9fd8-3355f169befb",
+   "id": "c8107823-ad7c-4804-9ffc-fa9777b52f1e",
    "metadata": {},
    "outputs": [],
    "source": [
-    "config = SourceDetectionTask.ConfigClass()\n",
-    "config.thresholdValue = 10\n",
-    "config.thresholdType = \"stdev\"\n",
-    "sourceDetectionTask = SourceDetectionTask(schema=schema, config=config)"
+    "simpleButler.registry.getCollectionChain(my_outputCollection)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "3d7949ad-d7df-4e7b-a3ac-6ea39aa2dd2f",
+   "id": "4e4beb42-518d-42f4-aa41-dc68c751351f",
    "metadata": {},
    "source": [
-    "No special configurations are used for the deblender in this example."
+    "## 2.2. Define your source detection pipeline"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "37f7d221-ba16-4390-92f9-ab32b0f7fa98",
+   "cell_type": "markdown",
+   "id": "755705e1-77a1-498a-b576-518e06271026",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "sourceDeblendTask = SourceDeblendTask(schema=schema)"
+    "Now for the pipeline definition. You will use a subset of the pipeline Tasks from the DP0.2 DRP pipline, which is defined by the `$DRP_PIPE_DIR/pipelines/LSSTCam-imSim/DRP-test-med-1.yaml` YAML file. Specifically, you'll need to run four \"steps\" from within the DP0.2 DRP pipeline: (1) `detection`, the initial detection of sources within your custom coadd (2) `mergeDetection`, which is needed to make possible downstream processing steps (3) `deblend`, deblending based on further analysis of the initial `detection` list (4) `measure`, which computes useful quantities, such as the photometric fluxes, given the deblended list of sources.\n",
```

Comment:
We're trying to move all NBs to the imperative mood; this is a relatively new addition to RTN-045 (S.2.6) but if we could update 09b now that'd be helpful. (I.e., narrative should be instructions without we/you)

Reply:
I see. I think maybe I had thought I knew what "imperative mood" meant but didn't. For some reason I thought the idea was to write with you/your rather than we/our, but I guess imperative mood actually means none of these.

Reply:
Done in [929cbe9](https://github.com/rubin-dp0/tutorial-notebooks/pull/161/commits/929cbe9e718fbadcfbf85e237505434d53a6dcfa).